core/cmd/shell_local.go

```diff
@@ -1036,3 +1098,43 @@ func insertFixtures(dbURL url.URL, pathToFixtures string) (err error) {
 	_, err = db.Exec(string(fixturesSQL))
 	return err
 }
+
+func getTablesContainingColumn(db *sqlx.DB, columnName string) ([]string, error) {
+	rows, err := db.Query("SELECT table_name FROM information_schema.tables WHERE table_schema='public'")
+	if err != nil {
+		return nil, err
+	}
+	defer rows.Close()
+
+	var tables []string
+	for rows.Next() {
+		var tableName string
+		if err = rows.Scan(&tableName); err != nil {
+			return nil, err
+		}
+		tables = append(tables, tableName)
+	}
+
+	var tablesWithColumn []string
+	for _, table := range tables {
+		has, err := hasColumn(db, table, columnName)
+		if err != nil {
+			return nil, err
+		}
+		if has {
+			tablesWithColumn = append(tablesWithColumn, table)
+		}
+	}
+	return tablesWithColumn, nil
+}
+
+func hasColumn(db *sqlx.DB, table, columnName string) (bool, error) {
+	query := fmt.Sprintf("SELECT column_name FROM information_schema.columns WHERE table_name='%s' AND column_name='%s'", table, columnName)
+	rows, err := db.Query(query)
```

Comment:
I thought that the linter normally complains about this style of sql construction. Can we pass the args through the method instead?

Reply:
Oh, and could this statement be combined with the first to avoid the for loop?

Reply:
Yeah, was actually easy to combine them, it can be further condensed to one query, but that will look a bit confusing and that performance doesn't matter much